ttbasic/ntsc.cpp

```diff
@@ -901,16 +904,15 @@ void ICACHE_RAM_ATTR VS23S010::MoveBlock(uint16_t x_src, uint16_t y_src,
 
 	// If the last move was a reverse one, we have to wait until it's
 	// finished before we can set the new addresses.
-	if (last_dir)
-		while (!blockFinished()) {
-		}
+	while (last_dir && !blockFinished()) {}
```

Comment:
This is not good. It makes it unclear what is actually being waited for, and suggests that `last_dir` may be variable within the scope of the `while()` loop.